### ISSUE TITLE
interpolation syntax on cloud-dns

### ISSUE THREAD

**Issue by yugandhar-btc** [terraform-google-modules/terraform-google-project-factory#728]
### TL;DR

when ran the _terraform init,_ it is Warning with **Interpolation-only expressions are deprecated** .

reference path:-
[dns_zones](https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit/blob/master/templates/tfengine/components/resources/dns_zones/main.tf)
[gke_clusters](https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit/blob/master/templates/tfengine/components/resources/gke_clusters/main.tf)

### Observed behavior

Warning: Interpolation-only expressions are deprecated

  on main.tf line 165, in module "example_dev":
 165:       records = ["${google_compute_global_address.ingress_static_ip.address}"]

Terraform 0.11 and earlier required all non-constant expressions to be
provided via interpolation syntax, but this pattern is now deprecated. To
silence this warning, remove the "${ sequence from the start and the }"
sequence from the end of this expression, leaving just the inner expression.

### Terraform Configuration

```hcl
module "example_qa" {
  source  = "terraform-google-modules/cloud-dns/google"
  version = "~> 3.0.0"

  name       = "example-qa"
  project_id = module.project.project_id
  domain     = "example-qa.example.com."
  type       = "public"

  recordsets = [
    {
      name    = "XXXXXXX"
      records = ["${google_compute_global_address.ingress_static_ip.address}"]
      ttl     = 30
      type    = "A"
    },
    {
      name    = "XXXXXXX"
      records = ["${google_compute_global_address.ingress_static_ip.address}"]
      ttl     = 30
      type    = "A"
    },
  ]

}
```

### Terraform Version

```sh
Terraform v0.14.0
```

**Comment by bharathkkb** [comment]
@yugandhar-btc I would recommend opening as issue in https://github.com/GoogleCloudPlatform/solution-acceleration-toolkit as this is something related to our modules. It's also worth nothing that this is just a warning and not an error.
